[MI-3708]: Added the test cases for attachment sync for MS Teams sync plugin. (#12)
Brightscout/mattermost-test-management: data/test-cases/plugins/ms-teams-sync/attachments-sync/Send_embedded_image.md

---
# (Required) Ensure all values are filled up
name: "Send embedded image"
status: Active
priority: Normal
folder: attachments-sync
authors: "@arush-vashishtha"
team_ownership: []
priority_p1_to_p4: P2 - Core Functions (Do core functions work?)

# (Optional)
location: null
component: null
tags: []
labels: []
tested_by_contributor: null

# (Optional) Test type and tools
cypress: null
detox: null
mmctl: null
playwright: null
rainforest: []
manual_test_environments: []

# Do not change
id: null
key: null
created_on: null
last_updated: null
case_hashed: null
steps_hashed: null
---

**Step 1**

1. Paste and send the embedded image in the text area in linked channel or DM/GM in MM.
2. Navigate to the linked channel or DM/GM in MS Teams.

**Step 2**

1. Paste and send the embedded image in the text area in linked channel or DM/GM in MS Teams.

> 2. Navigate to the linked channed or DM/GM in MM.

Reviewer: ```suggestion
2. Navigate to the linked channel or DM/GM in MM.
```

---
# (Required) Ensure all values are filled up
name: "Send embedded image"
status: Active
priority: Normal
folder: attachments-sync
authors: "@AayushChaudhary0001"
team_ownership: []
priority_p1_to_p4: P2 - Core Functions (Do core functions work?)

# (Optional)
location: null
component: null
tags: []
labels: []
tested_by_contributor: null

# (Optional) Test type and tools
cypress: null
detox: null
mmctl: null
playwright: null
rainforest: []
manual_test_environments: []

# Do not change
id: null
key: null
created_on: null
last_updated: null
case_hashed: null
steps_hashed: null
---

**Step 1**

> 1. Paste the embedded image in the text area of the channel or DM/GM in MM.
> 2. Click on send button in MM.

Reviewer 2: ```suggestion
1. Send an embedded image in the linked channel or DM/GM in MM.
```

3. Navigate to the linked channel or DM/GM in MS teams.

**Step 2**

1. Paste the embedded image in the text area of the channel or DM/GM in MS teams.

> 2. Click on send button in MS teams.

Reviewer 2: same

**Step 1**

> 1. Paste the embedded image in the text area and click on send button in linked channel or DM/GM in MM.

Reviewer 2: ```suggestion
1. Paste and send the embedded image in the linked channel or DM/GM in MM.
```

2. Navigate to the linked channel or DM/GM in MS Teams.

**Step 2**

> 1. Paste the embedded image in the text area and click on send button in linked channel or DM/GM in MS Teams.

Reviewer 2: same
